main.go

```diff
@@ -154,6 +154,7 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
 
+	daemonCommand.PersistentFlags().Bool(f.Service.NodeAutoRepair, false, "Enable Node Auto repair feature.")
```

Comment:
Why is the default `false` here (no auto repair), while it's `true` in the config?

Reply:
fixed